seqr/utils/elasticsearch/es_search.py

```diff
@@ -1048,7 +1048,7 @@ def _liftover_grch37_to_grch38():
         try:
             LIFTOVER_GRCH37_TO_GRCH38 = LiftOver('hg19', 'hg38')
         except Exception as e:
-            logger.warn('WARNING: Unable to set up liftover. {}'.format(e))
+            logger.error('WARNING: Unable to set up liftover. {}'.format(e))
```

Comment:
As above.  Should the WARNING be changed to ERROR?

Reply:
done